index.html

```diff
@@ -30,7 +30,7 @@ <h2 class="call-out out"> <a href="#project">check out my projects </a> </h2>
   <div class="projects-flex">
    <img class="project-title" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg" alt="project">
    <img class="project-title" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/map.jpg" alt="project">
-   <img class="project-title" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" alt="project">
+   <img class="project-title" id="project" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" alt="project">
    <img class="project-title" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tic-tac-toe.png" alt="project">
   </div>
   <div class="profile-link call-out">Show All</div>
```

Comment:
I would suggest to always allow a 2 space indentation in your code base 👍🏽

Reply:
- [ ] [REQUIRED] Nice Job phrasing your comments as a suggestion, and being polite. However, I kindly suggest you give reasons for the requested change.

:x: Wrong
I would suggest to always allow a 2 space indentation in your code base 👍🏽

:heavy_check_mark: Right
I would suggest to always allow a 2 space indentation in your codebase because it will make your code neat 👍🏽

Reply:
- Kindly edit your review as suggested by the previous reviewer to `I would suggest to always allow a 2 space indentation in your codebase because it will make your code neat 👍🏽`